controller/userController.js

```diff
@@ -0,0 +1,33 @@
+const File = require("../models/fileModel");
+const { v4 :uuidv4 }= require ('uuid');
+
+const fileUploader=async(req, res) => {
+      //validate
+      // console.log(req.file);
+      if (!req.file) {
+        return res.json({ error: "All fields are required" });
+      }
+    //   if(err) {
+    //     return res.status(500).send({ error: err.message }); 
+    //   }
+  
+      //store to database
+      const file = new File({
+        filename: req.file.filename,
+        uuid: uuidv4(),
+        path: req.file.path,
+        size: req.file.size,
+      });
+  
+      const response = await file.save().then((data)=>{
```

Comment:
Format this file, formatting is needed everywhere in the repo

Reply:
Formatting done